### DISCUSSION TITLE
Dashboard Filtering & Auto-Update Scheduling - v3.8.2

### DISCUSSION THREAD

**Discussion by alb3rtod** [chartbrew/chartbrew#250] [Ideas]
I have a database where I store the same type of data from different devices. I obtain this information every hour. In version 3.6.0, what I used to do in my dashboards was to create charts with the data shared by each device. To view the data from the specific device I wanted, I applied the device name as a filter in the 'Add filter' option on my dashboard and performed a 'Refresh charts' each time I changed the device name. This process is time-consuming because the tool queries the database each time before applying the device filter.

In version 3.8.2, I see that the implementation of Redis for scheduling the auto-update of charts does not apply the filter that I have set in the 'Add filter' option. It seems like it only updates the query data but doesn't apply the filter to the charts. Is this the expected behavior, or should it respect the filter set in 'Add filter'?

That being said, I find the auto-update of queries excellent. However, since it doesn't respect the filter you have set, is it possible to add an option to perform a 'Refresh filter' on the dashboards so that it only applies the filter to the charts without having to run a complete query on the database?

**Comment by razvanilin** [comment]
That seems like a bug in the new version. The filter should be applied automatically after a refresh. Will look into this and update 👌

**Comment by alb3rtod** [comment]
Resolved on commit: [e2c8117](https://github.com/chartbrew/chartbrew/commit/e2c81175ac220a7c66c9c68e7f767a918e2f5b8b)
